stock_picking_auto_revert/models/stock_picking.py

```diff
@@ -54,10 +43,15 @@ def action_revert_recreate(self):
         result["res_id"] = new_pick.id
         return result
 
-    def check_return_quantities(self, return_wiz, pick, msg):
+    def _check_return_quantities(self, return_wiz, pick):
+        msg = _(
+            "Too bad. This picking cannot be returned because the products "
+            "are not available in the destination location"
+        )
         for rm in return_wiz.product_return_moves:
-            sm = pick.move_lines.filtered(
-                lambda x: x.product_id.id == rm.product_id.id and x.state == "done"
+            sm = pick.move_ids.filtered(
+                lambda x, rm=rm: x.product_id.id == rm.product_id.id
+                and x.state == "done"
             )
             if rm.quantity < sum(sm.mapped("product_uom_qty")):
```

Comment:
Shouldn't we use `float_compare()`?

Reply:
This is the preexisting logic, the change is only to fix a lint error.
And this is an ID comparison, so I don't think float_compare() is helpful here.